tutorial-701/acquia-pipelines.yaml

```diff
@@ -0,0 +1,32 @@
+version: 1.0.0
+
+variables:
+  global:
+    # Paste your n3_key from ~/.acquia/pipelines/credentials on the next line.
```

Comment:
I think I'd rather ask people to create a new one. Or at least suggest creating a new one in addition.